lab-server/src/user/user.seeder.ts

```diff
@@ -0,0 +1,31 @@
+import { Connection } from 'typeorm';
+import { Factory, Seeder } from 'typeorm-seeding';
+
+import { User } from './entities/user.entity';
+
+export default class CreateUsers implements Seeder {
+    private mockData: User[] = [
+        {
+            createdAt: new Date(),
+            updatedAt: new Date(),
+            id: 1,
+            sub: '6314b9fc-2054-4637-be77-9e0cc48c186f',
+            allelesFile: '204753010001_R01C01.json',
+        },
+        {
+            createdAt: new Date(),
+            updatedAt: new Date(),
+            id: 2,
+            sub: '4beb7675-e415-4580-bdcb-75b63ca14766',
+            allelesFile: '204753010001_R02C01.json',
+        },
+    ];
+
+    public async run(_: Factory, connection: Connection): Promise<void> {
+        await Promise.all(
+            this.mockData.map(async (x) => {
+                await connection.getRepository(User).save(x);
+            }),
+        );
```

Comment:
Could be improved using a batch write. Does not need to be changed though (just a suggestion)

Reply:
I did change it